lib/swagger/jsonLoader.ts

```diff
@@ -106,11 +115,16 @@ export class JsonLoader implements Loader<Json> {
     return refObj;
   }
 
+  public getRealPath(mockName: string): string {
+    return this.mockNameMap[mockName];
+  }
+
   private async resolveRef(
     object: Json,
     pathArr: string[],
     rootObject: Json,
-    relativeFilePath: string
+    relativeFilePath: string,
+    skipResolveChildRef: boolean
```

Comment:
why don't make this as optional?

Reply:
It's a private function and I want to make it explicit to avoid bugs.